src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,368 @@
+      "filesystem": "本地文件"
+    }
+  },
+  "firmware": {
+    "current": "现有固件版本: v{version}",
+    "downloading_firmware": "下载固件中...",
+    "load_firmware": "上传至SD卡",
+    "loaded_firmware": "固件 {version} 已经上传至SD卡",
+    "loaded_firmware_info": "插入SD卡并启动Pocket完成新固件安装",
+    "no_firmware_loaded": "未发现SD卡上有待安装的固件",
+    "options": {
+      "latest": "v{version} (latest)",
+      "older": "v{version} ({date, date, short})",
+      "previous_versions": "上一版本"
+    },
+    "release_notes": {
+      "from": "来自 ",
+      "title": "v{version} 更新记录:"
+    },
+    "remove_button": "移除"
+  },
+  "games": {
+    "controls": {
+      "category": "类型",
+      "clean_files": "清理文件",
+      "instance_json": "实例配置JSON", 
+      "refresh": "刷新",
+      "search": "搜索"
+    },
+    "item": {
+      "game_count": "共找到{count, plural, =0 {0个游戏} one {# 个游戏} other {# 个游戏}}"
+    }
+  },
+  "image_editor": {
+    "cancel": "取消",
+    "import_image": "导入图像",
+    "remove": "移除",
+    "reset": "重置",
+    "rotate": "旋转",
+    "save": "保存",
+    "scale": "缩放"
+  },
+  "instance_json": {
+    "available_for": "以下核心可以自动生成实例配置JSON文件:", 
+    "close_button": "关闭",
+    "skipped_file": "跳过 {file_name}",
+    "title": "生成实例配置JSON文件", 
+    "wrote_file": "写入 {file_name}"
+  },
+  "layout": {
+    "cores": "核心",
+    "fetch": "同步",
+    "firmware": "固件",
+    "games": "游戏",
+    "platforms": "平台",
+    "pocket_sync": "Pocket Sync",
+    "save_states": "即时存档",
+    "saves": "存档",
+    "screenshots": "截屏",
+    "settings": "设置"
+  },
+  "mister_sync": {
+    "controls": {
+      "back": "退出",
+      "search": "搜索存档"
+    },
+    "login": {
+      "connect": "连接",
+      "host": "主机名/IP地址:",
+      "password": "密码:",
+      "username": "用户名:"
+    },
+    "mister": "MiSTer",
+    "not_found": "未找到",
+    "pocket": "Pocket",
+    "tip_1": "MiSTer FTP 功能必须打开",
+    "tip_2": "在传输前请确认存档文件兼容Pocket和MiSTer",
+    "tip_3": "存档匹配要求Pocket和MiSTer上的游戏文档名称完全一致",
+    "unsupported": "尚未支持{platform}平台\n(如果这个平台应该被支持，请在github上反馈)"
+  },
+  "news_feed": {
+    "last_updated": "最新更新 <1></1>",
+    "updates_from": "更新信息来自 <1>OpenFPGA Cores Inventory</1>"
+  },
+  "platform_info": {
+    "controls": {
+      "back": "返回列表",
+      "data_packs": "数据包",
+      "image_packs": "图像包"
+    },
+    "cores": "核心",
+    "edit": "编辑"
+  },
+  "platforms": {
+    "cancel": "取消",
+    "controls": {
+      "data_packs": "数据包",
+      "image_packs": "图像包",
+      "search": "搜索"
+    },
+    "data_packs": {
+      "apply_changes": "共执行了 {count, plural, =0 {0项修改} one {1项修改} other {# 项修改}}",
+      "close": "关闭",
+      "current": "现有"
+    },
+    "delete_unused": "删除 {count, plural, one {# 个未被使用的平台} other {# 个未被使用的平台}}?",
+    "edit": "编辑",
+    "image_packs": {
+      "apply_changes": "共执行了 {count, plural, =0 {0项修改} one {1项修改} other {# 项修改}}",
+      "close": "关闭",
+      "current": "现有"
+    },
+    "save": "保存"
+  },
+  "progress": {
+    "remaining_time": "剩余时间 (approx): {remainingTime}"
```

Comment:
Should the `(approx)` be translated? - shortening of "approximately" to show that the remaining time is just a guess (it's based on what's been downloaded / installed already)

Reply:
I missed this. Done